.kokoro/presubmit/integration.cfg

```diff
@@ -6,11 +6,6 @@ env_vars: {
   value: "gcr.io/cloud-devrel-kokoro-resources/java8"
 }
 
-env_vars: {
-    key: "INTEGRATION_TEST_ARGS"
-    value: "-P bigtable-emulator-it,bigtable-prod-it -Dbigtable.project=gcloud-devel -Dbigtable.instance=google-cloud-bigtable -Dbigtable.table=integration-tests"
-}
-
```

Comment:
I don't think that right

Reply:
For templates, there's a list of files to exclude in the `synth.py` file. We can add this file to it, but you wont be notified in the future.